core/src/test/java/io/questdb/test/cutlass/http/IODispatcherTest.java

```diff
@@ -8846,4 +8848,8 @@ public void run() {
     static class Status {
         boolean valid;
     }
+
+    static {
+        LogFactory.configureSync();
```

Comment:
Is this set on purpose or it's a debug leftover?

Reply:
yeah, removed here https://github.com/questdb/questdb/pull/3399/commits/64d685ac2b97d9b19cbe51543369ab292de3088e